### ISSUE TITLE
Parser doesn't recognise Oracle ( + ) operator, if it contains whitespace

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#10675]
Unfortunately, the Oracle `(+)` operator for outer joins is not a single token. It consists of 3 tokens `(` `+` `)`. As such, this is valid Oracle SQL:

```sql
select cd, count(t_book.id)
from t_language, t_book
where t_language.id = t_book.language_id(
 +
)
group by cd
```

To produce:

```
CD|COUNT(T_BOOK.ID)|
--|----------------|
fr|               0|
pt|               1|
de|               1|
en|               2|
```

The parser currently doesn't correctly parse this, but produces this error:

> Unknown function: [4:2] ...t_book
> where t_language.id = t_book.language_id(
>  [*]+
> )
> group by cd

**Comment by lukaseder** [comment]
Related: #10670

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.14.0. Backport scheduled for 3.13.5 (#10676)
